apps/builder/.gitignore

```diff
@@ -10,3 +10,4 @@
 /api/_assets
 /public/uploads
 /public/s/uploads
+/uploads
```

Comment:
this should no be used as upload folder. It's not accessible from app

Reply:
FILE_UPLOAD_PATH="uploads"
And I had that files uploaded into uploads dir